media/src/main/java/io/wcm/handler/media/util/ImageUtils.java

```diff
@@ -0,0 +1,96 @@
+/* Copyright (c) pro!vision GmbH. All rights reserved. */
```

Comment:
please apply the usual wcm.io apache license file header (with 2020 as date)

Reply:
fixed